### PR TITLE
feat(readme): add tmux-nord-plusplus

### PR DESCRIPTION
**Pull request by Wabri**
Add tpm plugin https://github.com/Wabri/tmux-nord-plusplus

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,4 +86,5 @@ A list of tmux plugins.
 - [tmux-colors-solarized](https://github.com/seebi/tmux-colors-solarized) - A solarizes theme for tmux.
 - [tmux-colours-superhero](https://github.com/leighmcculloch/tmux-colours-superhero) - A superhero themed tmux colour theme.
 - [tmux-dark-notify](https://github.com/erikw/tmux-dark-notify) - Make tmux's theme follow macOS dark/light mode.
+- [tmux-nord-plusplus](https://github.com/Wabri/tmux-nord-plusplus) - An arctic, north-bluish clean and elegant tmux color theme with support for battery & pomodoro.
```
Comment (oalders) [inline README.md:89]: It would probably make sense to list the project this is forked from here as well: https://github.com/nordtheme/tmux

Comment (Wabri) [reply README.md:90]: Yes you are right

File: README.md [hunk 2]
```diff
@@ -86,5 +86,6 @@ A list of tmux plugins.
 - [tmux-colors-solarized](https://github.com/seebi/tmux-colors-solarized) - A solarizes theme for tmux.
 - [tmux-colours-superhero](https://github.com/leighmcculloch/tmux-colours-superhero) - A superhero themed tmux colour theme.
 - [tmux-dark-notify](https://github.com/erikw/tmux-dark-notify) - Make tmux's theme follow macOS dark/light mode.
+- [tmux-nord](https://github.com/nordtheme/tmux) - An arctic, north-bluish clean and elegant tmux color theme
```
Comment (oalders) [inline README.md:89]: 👍🏻

**Comment by Wabri** [comment]
There are something that I miss that I need to add?

**Comment by bruno-** [comment]
Thanks!

**Pull request merged**